### ISSUE TITLE
aria-required should be removed if required is already set to be w3c compliant

### ISSUE THREAD

**Issue by nogueiraludovic** [in2code-de/powermail#1353]
Hello,

We are trying to make our website w3c compliant. For that we need to remove the "aria-required" if "required" is already there. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-required
Aria-required is only used for div or other HTML tag that have a specific role defined

You need to remove the multiple iteration of aria-required here: https://github.com/in2code-de/powermail/blob/master/Classes/ViewHelpers/Validation/ValidationDataAttributeViewHelper.php#L84
